src/components/Displaybooks.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Book from './Book';
+import { fetchBooks } from '../features/book/Bookslice';
+
+function Displaybooks() {
+  const { books } = useSelector((state) => state.book);
+  //   const { categories } = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
```

Comment:
- [ ] Let's try not to have commented unused code in our codebase as they make our code dirty and not professional. Kindly remove all unused codes from your codebase 👍🏽